### PR TITLE
feat: Add time stamp column to sync mode

### PR DESCRIPTION
**Pull request by BenFidge**
Fixes #2762

As per Lukas request, `Sync` mode now outputs `TimeStamp` field by default

### CODE REVIEW

**Review by Kieranties — CHANGES_REQUESTED** [review]
@BenFidge For the default sync mode, with the new column being added - what will happen for existing stream tables from 3.2.4? I assume it will blow up?

We need to make sure that existing stream tables can still be written to, ideally without having to re-process all the data.

File: GitVersion.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # GitVersion.yml
 mode: ContinuousDeployment
-next-version: 3.2
+next-version: 3.3
```
[the remark this answers is not in the document]
Comment (BenFidge) [reply GitVersion.yml]: Not sure how that happened!

File: docs/3.2.5-release-notes.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ### Features
 + Supports Streams multi modes (Event Stream, Sync)
++ Sync mode now adds `TimeStamp` field by default
```
[the remark this answers is not in the document]
Comment (Kieranties) [reply docs/3.2.5-release-notes.md]: @BenFidge We do need a release note here - you are changing the default implementation.  Please add the line as suggested

**Comment by BenFidge** [comment]
Relies on main repo [PR 2774](https://github.com/CluedIn-io/CluedIn/pull/2774)

**Pull request merged**